rsyslog/Dockerfile

```diff
@@ -22,12 +22,14 @@ RUN cd /vendored_src && \
 
 ADD go/ /go/
 RUN cd /go/src/github.com/soundcloud/rsyslog_exporter ; GOPATH=/go go build
+RUN cd /go/src/github.com/mmexternal/undefined_field ; GOPATH=/go go build
```

Comment:
hmm - then put the code in the `rsyslog/undefined_field` sub-directory
```
ADD undefined_field/ /undefined_field/
RUN cd /undefined_field/ ; go build
```
not sure what to use for `GOPATH` here - not sure it matters.